locations/spiders/marathon_petroleum_us.py

```diff
@@ -0,0 +1,85 @@
+                        fuel_types_matched.append(Fuel.DIESEL)
+                    case "DSL2" | "DIESEL 2" | "#2 DIESEL" | "DSL #2" | "DSL#2" | "DIESEL #2" | "DIESEL2" | "DSL 2 PREM" | "DSL 2" | "#2 DSL" | "DIE 2" | "DIESL2" | "REG DSL #2" | "DIESEL #2`" | "DIESEL 2 N" | "DIESEL 2 S":
+                        fuel_types_matched.append(Fuel.COLD_WEATHER_DIESEL)
+                    case "TRUCK DIESEL" | "TRK DIESEL" | "TRUCK DSL" | "TUCK DSL" | "TDSL" | "TRKDSL" | "TRK DSL":
+                        fuel_types_matched.append(Fuel.HGV_DIESEL)
+                    case "DSL TAX EX":
+                        fuel_types_matched.append(Fuel.UNTAXED_DIESEL)
+                    case "BIODSL 15%":
+                        fuel_types_matched.append(Fuel.BIODIESEL)
+                    case "UNLD" | "REGUNL" | "UNLEAD" | "UNLEADED" | "UNLD1" | "REGULAR" | "REG" | "REG1" | "87" | "UNLEADED FUEL" | "REG 87" | "UNLD87" | "NO ETH 87" | "UNLD 87" | "UNL" | "RUL" | "RUNL" | "REGULAR UNLEADED" | "REG UNLEADED" | "UNLEADED1" | "UNLEADED2" | "REGULAR UL" | "UNLEAD REG" | "UNLEAD 1" | "UNLEAD 2" | "UNLEAD3" | "87 OXY" | "CONV87" | "RNL" | "REGUALR" | "UNLD2" | "UNLD3" | "87NO ETH" | "87-ETHFREE" | "UNLEAD1" | "UNLEAD2" | "ULEADED" | "UNL REG" | "EC UNLD" | "EC UNLEAD" | "UNLEADED 1" | "UNLEADED 2" | "REGULAR1" | "REC FUEL" | "REGU" | "REG NL" | "UNLEADED 2" | "UNLED" | "REG 1" | "REG 2" | "UNLEADED87" | "UNLEADED REG" | "UNLEAD 87" | "STRA87" | "UNL REGULAR" | "REGULAR2" | "REG2" | "REGULAR NL" | "UNLD 1" | "CONV UNLD" | "NO UNL" | "REGNL" | "REG UNLEAD" | "87 NON" | "OXY 87" | "OXY 87 OCT" | "REGULA" | "REGULAR 1" | "REGULAR 2" | "REGUL" | "REGULR" | "REG UNLD" | "UNDLEAD" | "UNLD/REG" | "UNL E0" | "UNLEADE" | "UNLEAD ROA" | "UNLED FUEL" | "87A" | "87B" | "NOLEAD 87 OXY" | "REG NO ETH" | "REG NOLEAD" | "REGULAR 2" | "REGULAR BLD" | "REG UL" | "REGULR" | "REGUNLD" | "REG UNL" | "REG UN" | "UNLD 2" | "UNLD REG" | "UNLD/REG" | "NON UNL" | "87 CON" | "E0 UNL" | "CONV REG" | "CONV U/L" | "CONV UNL" | "NON E REG" | "NON ETHANOL REG" | "UNLD REG." | "UNLEAD87":
+                        fuel_types_matched.append(Fuel.OCTANE_87)
+                    case "UNLD88":
+                        fuel_types_matched.append(Fuel.OCTANE_88)
```

Comment:
Looks like `Fuel.OCTANE_88` might need to get added

Reply:
Thanks, I lost that change to categories.py with an accidental git command. Added 88 and also 96 that was missing (completing the full list of 87 through 100 inclusive)

Reply:
Thanks for going through all the fuel types. That's a lot of distinct values!